File: Binary_Decimal.java
import java.util.Scanner;

public class BinaryConversion {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the number to find out its binary form: ");
        int a = scanner.nextInt();

        int ans = 0;
        int count = 0;
        while (a > 0) {
            int r = a % 10;
            ans += r * Math.pow(2, count);
            a /= 10;
            count++;
        }

        System.out.println("Binary form: " + ans);
        scanner.close();
    }
}

File: Binary_Decimal.cpp
#include <iostream>
#include <cmath>
using namespace std;
int main() {
    int a ;
    int ans =0;
    int result = 0;
    int rem;
    int r;
    int count =0;
    cout<<"enter the number to find out the its binary form = ";
    cin>>a;
    
    while(a>0){
        r=a%10;   
        ans = ans+r*pow(2,count);
       a=a/10;
       count++;
    }
    
    
   cout<<ans;
        
    return 0;
}
